public/app/features/dashboard-scene/sharing/ShareLinkTab.tsx

```diff
@@ -60,6 +61,7 @@ export class ShareLinkTab extends SceneObjectBase<ShareLinkTabState> {
 
     if (panel) {
       urlParamsUpdate.viewPanel = panel.state.key;
+      urlParamsUpdate.panelId = getPanelIdForVizPanel(panel);
```

Comment:
I modified this since the panelId is needed for the rendering plugin to work

Reply:
this breaks a few tests though. probably should keep the panelId only for rendering